components/script/dom/extendableevent.rs

```diff
@@ -41,9 +42,10 @@ impl ExtendableEvent {
         ev
     }
 
-    pub fn Constructor(global: &GlobalScope,
+    pub fn Constructor(worker: &ServiceWorkerGlobalScope,
                        type_: DOMString,
                        init: &ExtendableEventBinding::ExtendableEventInit) -> Fallible<Root<ExtendableEvent>> {
+        let global = worker.upcast::<GlobalScope>();
```

Comment:
I think this should be `&ServiceWorkerGlobalScope`, based on `ExtendableEvent.webidl` Updated!